h2o-docs/src/product/data-science/algo-params/missing_values_handling.rst

```diff
@@ -17,7 +17,7 @@ The fewer the NA values in your training data, the better. Always check degrees
 Related Parameters
```

Comment:
Also need to update the description because a new option was added to missing_values_handling. (See https://github.com/h2oai/h2o-3/pull/3821/files#diff-1bc6d9a43cd27577320aabef1d6ded38L117)

"In H2O, the Deep Learning algorithm will either skip or mean-impute rows with NA values. The GLM algorithm can skip or mean-impute rows with NA values, or you can use `plug_values`, which allows you to then specify a single-row frame that contains values that should be used to impute missing values of the training/validation frame. Both algorithms default to MeanImputation. ..."